### PR TITLE
IN-51: Add api call for editing user

### PR DESCRIPTION
**Pull request by spanastasia**

### CODE REVIEW

File: INTITA/Networking/ApiURL.swift [hunk 1]
```diff
@@ -100,6 +108,8 @@ enum ApiURL {
             return "currentUser"
         case .logout:
             return nil
+        case .editUser:
+            return nil
```
Comment (oleksiikolomiietssnapp) [inline INTITA/Networking/ApiURL.swift:112]: Please add filename(find the file in chat)

File: INTITA/Networking/Services/Authorization.swift [hunk 2]
```diff
@@ -23,6 +22,17 @@ enum HTTPType {
...
+    var editService: EditUserProtocol {
+        switch self {
+        case .real:
+            return EditUserReal.shared
+        case .mock:
+            return EditUserMock()
+        case .fail:
+            return EditUserFailing()
+        }
+    }
```
Comment (oleksiikolomiietssnapp) [inline INTITA/Networking/Services/Authorization.swift:35]: I think it's unnecessary.

Comment (oleksiikolomiietssnapp) [reply INTITA/Networking/Services/Authorization.swift:35]: It is better to add 
`func editUserInfo(newUser: EditingUser, completion: @escaping (Error?) -> Void) {`

**Review by oleksiikolomiietssnapp — APPROVED** [review]

File: INTITA/Networking/ApiURL.swift (hunk 1 above)
Comment (spanastasia) [reply INTITA/Networking/ApiURL.swift:112]: Done.

File: INTITA/Networking/Services/Authorization.swift (hunk 2 above)
Comment (spanastasia) [reply INTITA/Networking/Services/Authorization.swift:35]: Done.

**Review by oleksiikolomiietssnapp — APPROVED** [review]

**Review by Viacheslav83 — APPROVED** [review]

**Pull request merged**